### PR TITLE
Edge Control - Product Page updates

### PR DESCRIPTION
**Pull request by jcarolinares**
## What This PR Changes
- New  button for the user manual, old getting started hidden, tutorials sorted by importance

## Contribution Guidelines
- [x] I confirm that I have read the [contribution guidelines](https://github.com/arduino/docs-content/tree/main/contribution-templates) and comply with them.

### CODE REVIEW

File: content/hardware/05.pro-solutions/solutions-and-kits/edge-control/tutorials/04.motorized-ball-valve/content.md [hunk 1]
```diff
@@ -9,60 +9,60 @@ author: Ernesto E. Lopez, Lenard George Swamy
...
 
-Before you attach the wires to the board you need to ensure that the Phoenix connectors are in place  
+Before you attach the wires to the board you need to ensure that the Phoenix connectors are in place.
 
 **The Latching Circuitry**
-A Latching Circuit has 2 inputs for Positive(P) and Negative(N) signals and 1 output. Latches allows you to store the previous state of the signal that passes through either of the pins and maintains the state until you pass a new signal. Signal passed through the positive and the negative pins will also determine the direction of the signal to the valve. The Edge Control board comes with 8 pairs of Latches each labeled from 1N:1P up to 8N:8P. 
+A Latching Circuit has 2 inputs for Positive(P) and Negative(N) signals and 1 output. Latches allows you to store the previous state of the signal that passes through either of the pins and maintains the state until you pass a new signal. Signal passed through the positive and the negative pins will also determine the direction of the signal to the valve. The Edge Control board comes with 8 pairs of Latches each labeled from 1N:1P up to 8N:8P.
```
Comment (g-castronovo) [inline content/hardware/05.pro-solutions/solutions-and-kits/edge-control/tutorials/04.motorized-ball-valve/content.md:43]:
```suggestion
A Latching Circuit has 2 inputs for Positive(P) and Negative(N) signals and 1 output. Latches allow you to store the previous state of the signal that passes through either of the pins and maintains the state until you pass a new signal. Signal passed through the positive and the negative pins will also determine the direction of the signal to the valve. The Edge Control board comes with 8 pairs of Latches each labeled from 1N:1P up to 8N:8P.
```

File: content/hardware/05.pro-solutions/solutions-and-kits/edge-control/tutorials/04.motorized-ball-valve/content.md [hunk 2]
```diff
@@ -104,15 +104,15 @@ void loop(){
 }
 ```
 
-### 3. Connecting To A Power Source 
+### 3. Connecting To A Power Source
 
 The valves require a power supply of 9 - 12 V and you can either use a regular power supply or a 3 cell LiPo battery to provide the required voltage. Power sources can be connected to the relay ports on the edge control board, ensure that you have a source that can provide 9 - 12 V of power for both the valve and the board. Connect two jumper wires to the **GND** and **B** pins of the **Relay ports**.
```
Comment (g-castronovo) [inline content/hardware/05.pro-solutions/solutions-and-kits/edge-control/tutorials/04.motorized-ball-valve/content.md:109]:
```suggestion
The valves require a power supply of 9 - 12 V and you can either use a regular power supply or a 3 cell LiPo battery to provide the required voltage. Power sources can be connected to the relay ports on the Edge Control board, ensure that you have a source that can provide 9 - 12 V of power for both the valve and the board. Connect two jumper wires to the **GND** and **B** pins of the **Relay ports**.
```

File: content/hardware/05.pro-solutions/solutions-and-kits/edge-control/tutorials/04.motorized-ball-valve/content.md [hunk 3]
```diff
@@ -9,60 +9,60 @@ author: Ernesto E. Lopez, Lenard George Swamy
...
 - [Arduino Edge control board](https://store.arduino.cc/edge-control)
-- 1x [US Solid Motorised Ball Valve (9 - 24 V)](https://ussolid.com/u-s-solid-motorized-ball-valve-1-2-brass-electrical-ball-valve-with-full-port-9-24-v-ac-dc-3-wire-setup.html) (or compatible)
-- External power source: 12V battery (LiPo / SLA) or power supply 
+- 1x [US Solid Motorized Ball Valve (9 - 24 V)](https://ussolid.com/u-s-solid-motorized-ball-valve-1-2-brass-electrical-ball-valve-with-full-port-9-24-v-ac-dc-3-wire-setup.html) (or compatible)
+- External power source: 12V battery (LiPo / SLA) or power supply
 - 1x Micro USB cable
 - Arduino IDE 1.8.10+
 - 2x Phoenix Connectors
-- 2x Jumper cables  
+- 2x Jumper cables
```
Comment (g-castronovo) [inline content/hardware/05.pro-solutions/solutions-and-kits/edge-control/tutorials/04.motorized-ball-valve/content.md:30]:
```suggestion
- 2x Jumper wires
```

**Review by g-castronovo — APPROVED** [review]
Just some suggestions.
Noticed in the body of the content we use Jumper wires > jumper cables

File: content/hardware/05.pro-solutions/solutions-and-kits/edge-control/tutorials/04.motorized-ball-valve/content.md (hunk 1 above)
Comment (jcarolinares) [reply content/hardware/05.pro-solutions/solutions-and-kits/edge-control/tutorials/04.motorized-ball-valve/content.md:43]: thanks!

**Pull request merged**